.github/workflows/build.yml

```diff
@@ -0,0 +1,75 @@
+name: Build
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+env:
+  OPENSSL_INSTALL: 'C:\OpenSSL-Win32'
+
+jobs:
+  cache-openssl:
+    runs-on: windows-2019
+    steps:
+      - name: Check if OpenSSL installer is cached
+        id: check-cache
+        uses: actions/cache@v4
+        with:
+          path: Win32OpenSSL-3_0_1.exe
+          key: '${{ runner.os }}-openssl-installer'
+
+      - name: Download and extract OpenSSL
+        if: steps.check-cache.outputs.cache-hit != 'true'
+        run: |
+          curl -o OpenSSL-Win32.zip "https://files.catbox.moe/xtfk7t.zip"
```

Comment:
When I put out a release for OpenSSL for MSVC (at https://github.com/DiscordMessenger/openssl), please use that instead.

Reply:
.lib files are missing from the release.

Reply:
> ~.lib files are missing from the release.~

Yes, those were intended for MinGW only. I just added the MSVC library files.